### ISSUE TITLE
`@stylistic/function-parentheses-newline-inside` fix takes the closing parenthesis into an inline comment

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#113]
Found while fixing #100, by sweeping every rule that writes a declaration's value back through `setDeclarationValue`. The same shape as #63, #87, #88 and #89: an inline comment ends only with a line break, so whatever the fix pulls onto its line is commented out. Here it is the closing parenthesis of the function.

### What minimal example or steps are needed to reproduce the bug?

```less
.a {
  transform: translate(
    1px, 2px // c
  );
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "customSyntax": "postcss-less",
  "plugins": "@stylistic/stylelint-plugin",
  "rules": {
    "@stylistic/function-parentheses-newline-inside": "never-multi-line"
  }
}
```

### What did you expect to happen?

The comment runs to the end of its line, so the closing parenthesis has nowhere to go: `never-multi-line` cannot be satisfied without commenting it out. The fix should leave the declaration alone and let the warning stand.

### What actually happened?

```less
.a {
  transform: translate(1px, 2px // c);
}
```

The closing parenthesis and the semicolon are now inside the comment, so the declaration ends nowhere and the value is left open. No warning survives the run: `--fix` reports a clean pass on a file it has just broken.

`postcss-less` reads the output back without complaining — it keeps a `//` in a value as ordinary text, which is what #32 is about — so nothing downstream of Stylelint catches this either. Less itself does not: for it the line ends at the double slash.

The same fixture under `postcss-scss` is spared for the time being, and only by accident: that syntax keeps a second copy of a value carrying an inline comment and prints it, while the fix is written into the copy it does not print, so the edit is dropped along the way. Whoever repairs that dropping — the value written where the syntax reads it — hands this rule the same corruption under SCSS.

The cause is at [`lib/rules/function-parentheses-newline-inside/index.js:109-115`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/function-parentheses-newline-inside/index.js#L109-L115), whose `fixAfterForNever` ([`:254`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/function-parentheses-newline-inside/index.js#L254)) empties the whitespace in front of the closing parenthesis without asking what stands in front of that whitespace.

### Do you have a proposal to fix the bug?

The shape the cluster settled on: where the text the fix would cut into ends with an inline comment, report the problem and write nothing. `endsWithInlineComment` in `lib/utils/` already answers that question, and it is the text between the last argument and the closing parenthesis that has to be put to it.

`always` and `always-multi-line` need nothing: they add a line break rather than take one away, and a line break is precisely what closes the comment.
